src/Process/LayeredMaterialNodeProcessing.js

```diff
@@ -289,7 +294,7 @@ export function updateLayeredMaterialNodeElevation(context, layer, node, force)
                 terrain.texture.flipY = false;
                 terrain.texture.needsUpdate = true;
             }
-            node.setTextureElevation(terrain);
+            if (terrain.texture.image.data[0] && terrain.texture.image.data[terrain.texture.image.data.length - 1] > -99999) node.setTextureElevation(terrain);
```

Comment:
Hmm I'm not really a fan of this proposal. I've seen some place where data[0] was valid but data[15] wasn't for instance.

Also have you considered this alternative: allow several textures layers?
They wouldn't blend their values like colors layers but instead the first elevation texture with a valid value would be used.
So in your case, you would declare `IGN_MNT_HIGHRES` as the first elevation layer, and `MNT_WORLD_SRTM3` as the second one and it would act as a fallback when `HIGH_RES` gives no value.

WDYT?